code/modules/mob/dead/observer/observer.dm

```diff
@@ -87,7 +87,7 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	real_name = name
 
 	//starts ghosts off with all HUDs.
-	toggle_all_huds_on()
+	toggle_all_huds_on(body)
```

Comment:
`body` will be null if the admin joins as an observer, so they won't get antaghud added. All the other states seem to be working fine.

Reply:
God how did I miss that.